packages/translations/locales/en/settings.json

```
@@ -173,6 +173,112 @@
+      "fullTime": "Full-Time",
+      "fullTimeDesc": "Available for full-time positions (40h/week)",
+      "partTime": "Part-Time",
+      "partTimeDesc": "Available for part-time positions",
+      "customSchedule": "Replacement/Substitute",
+      "customScheduleDesc": "Available for replacement work with specific hours"
+    },
+    "weeklySchedule": {
+      "title": "Weekly Schedule",
+      "subtitle": "Set your regular availability for each day",
+      "days": {
+        "sunday": "Sunday",
```

Biome (2.1.2)

[error] The key days was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)